e2e/terraform/provision-infra/variables.tf

```diff
@@ -103,6 +103,12 @@ variable "aws_kms_alias" {
 # provide a list of builds to override the values of nomad_sha, nomad_version,
 # or nomad_local_binary. Most of the time you can ignore these variables!
 
+variable "nomad_local_binary_server_unique" {
+  description = "A nomad local binary paths to deploy to servers, to override nomad_local_binary"
```

Comment:
I'm not sure I understand what this variable is supposed to do (what's "unique" mean here?), and the description matches `nomad_local_binary_server` exactly. It seems like we're overriding `nomad_local_binary_server` but I can't tell why. Shouldn't Enos set `nomad_local_binary_server` to make sure we're using the Linux binaries for the server instead?

Reply:
because nomad_local_binary_server is an array and we would need to pass the same path 3 times, but enos does not accept functions, so it would have to come from the previous step as a list or we would have to add an extra module to build the list

Reply:
Ah, ok... honestly I'm not sure that array has _ever_ been used to have different values, so I wonder if it would be worth refactoring to just have a single override. But we can do that in a separate PR if you want.

Reply:
Im guessing at some point someone wanted to test compatibility of different versions?

Reply:
Yeah. I think I wrote that? :blush: But we really got away from running E2E manually, so there's probably a bunch of tiny design decisions that could be improved there.